pkg/run/install/install_dashboard_test.go

```diff
@@ -194,51 +194,51 @@ var _ = Describe("InstallDashboard", func() {
 
 var _ = Describe("GetInstalledDashboard", func() {
 	var (
-		fakeContext                context.Context
-		fakeClientWithHelmReleases client.WithWatch
-		fakeClientWithDeployments  client.WithWatch
-		blankClient                client.WithWatch
-		errorClient                ErroringFakeClient
+		fakeContext context.Context
+		// fakeClientWithHelmReleases client.WithWatch
+		fakeClientWithDeployments client.WithWatch
+		blankClient               client.WithWatch
+		errorClient               ErroringFakeClient
 	)
 
 	BeforeEach(func() {
 		fakeContext = context.Background()
 		scheme, err := kube.CreateScheme()
 		Expect(err).NotTo(HaveOccurred())
 
-		fakeClientWithHelmReleases = fake.NewClientBuilder().WithScheme(scheme).WithRuntimeObjects(helmReleaseFixtures...).Build()
+		// fakeClientWithHelmReleases = fake.NewClientBuilder().WithScheme(scheme).WithRuntimeObjects(helmReleaseFixtures...).Build()
 		fakeClientWithDeployments = fake.NewClientBuilder().WithScheme(scheme).WithRuntimeObjects(deploymentFixtures...).Build()
 		blankClient = fake.NewClientBuilder().WithScheme(scheme).Build()
 		errorClient = ErroringFakeClient{}
 	})
 
-	It("returns the oss dashboard type if the dashboard is installed with a helmrelease", func() {
-		dashboardType, dashboardName, err := GetInstalledDashboard(fakeContext, fakeClientWithHelmReleases, testNamespace, map[DashboardType]bool{
-			DashboardTypeOSS: true,
-		})
-		Expect(err).ToNot(HaveOccurred())
-		Expect(dashboardType).To(Equal(DashboardTypeOSS))
-		Expect(dashboardName).To(Equal("dashboard-2"))
-	})
-
-	It("returns the enterprise dashboard type if the dashboard is installed with a helmrelease", func() {
-		dashboardType, dashboardName, err := GetInstalledDashboard(fakeContext, fakeClientWithHelmReleases, testNamespace, map[DashboardType]bool{
-			DashboardTypeEnterprise: true,
-		})
-		Expect(err).ToNot(HaveOccurred())
-		Expect(dashboardType).To(Equal(DashboardTypeEnterprise))
-		Expect(dashboardName).To(Equal("dashboard-3"))
-	})
-
-	It("returns the enterprise dashboard type if both dashboards are installed with a helmrelease", func() {
-		dashboardType, dashboardName, err := GetInstalledDashboard(fakeContext, fakeClientWithHelmReleases, testNamespace, map[DashboardType]bool{
-			DashboardTypeOSS:        true,
-			DashboardTypeEnterprise: true,
-		})
-		Expect(err).ToNot(HaveOccurred())
-		Expect(dashboardType).To(Equal(DashboardTypeEnterprise))
-		Expect(dashboardName).To(Equal("dashboard-3"))
-	})
+	// It("returns the oss dashboard type if the dashboard is installed with a helmrelease", func() {
```

Comment:
@erikgb These seem to be the last remaining items to get this merged

Reply:
Nice @casibbald! Should I take a look tonight?